sdk/src/client/node_api/core/mod.rs

```diff
@@ -69,6 +98,40 @@ impl Client {
         Ok(outputs)
     }
 
+    /// Request outputs and their metadata by their output ID in parallel, ignoring failed requests
+    /// Useful to get data about spent outputs, that might not be pruned yet
+    pub async fn get_outputs_with_metadata_ignore_errors(
```

Comment:
i dont like this code duplication, we should make both call an internal method with an additional boolean parameter

Reply:
issue will be made separate from this